src/sidebar/util/annotations-by-user.ts

```diff
@@ -0,0 +1,42 @@
+import type { APIAnnotationData } from '../../types/api';
+import { isReply } from '../helpers/annotation-metadata';
+
+/**
+ * Details of a user and their annotations that are available to import or export.
+ */
+export type UserAnnotations = {
+  userid: string;
+  displayName: string;
+  annotations: APIAnnotationData[];
+};
+
+/**
+ * Generate an alphabetized list of authors and their importable/exportable
+ * annotations.
+ */
+export function annotationsByUser(
+  anns: APIAnnotationData[],
+  getDisplayName: (ann: APIAnnotationData) => string,
+): UserAnnotations[] {
```

Comment:
We should add some basic unit tests for this helper specifically. This may make it possible to simplify or eliminate indirect tests for this in `ImportAnnotations`/`ExportAnnotations`. For example we can test the sort order in the tests for this helper rather than in the component tests.